client/my-sites/importer/importing-pane.jsx

```diff
@@ -149,20 +149,20 @@ export const ImportingPane = React.createClass( {
 			);
 		}
 
-		return this.translate( 'Import complete!' );
+		return translate( 'Import complete!' );
 	},
 
 	getImportMessage( numResources ) {
 		if ( 0 === numResources ) {
-			return this.translate( 'Finishing up the import' );
+			return translate( 'Finishing up the import' );
 		}
 
-		return this.translate(
-			'Waiting on %(numResources)d resource to import',
-			'Waiting on %(numResources)d resources to import',
+		return translate(
+			'Waiting on %(numResources)s resource to import',
```

Comment:
Not yet, all importers use this pane and the same message, so Medium is still showing `Waiting on X resources...`.